### PR TITLE
junitTestResults is compulsory ever when publishing is off

### PR DESCRIPTION
**Pull request by shashwatgupta**
junitTestResults is compulsory fields even when Publish to VSO/TFS is turned off - Making them visible only when Publish to VSO for test result is turned on

### CODE REVIEW

File: Tasks/ANT/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 13
+        "Patch": 14
```
Comment (acesiddhu) [inline Tasks/ANT/task.json]: make sure u merge this with Prawal changes. both version and the windows thing he added

**Comment by acesiddhu** [comment]
rest is fine

File: Tasks/PublishTestResults/task.loc.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
         "JUnit": "JUnit",
         "VSTest": "VSTest",
         "NUnit": "NUnit",
-        "XUnit": "XUnit"
+        "XUnit": "XUnit V2"
```
Comment (AshwiniChalla) [inline Tasks/PublishTestResults/task.loc.json]: Hope is only auto generated and not your change

Comment (shashwatgupta) [inline Tasks/PublishTestResults/task.loc.json]: Yes auto generated

**Pull request merged**